src/App.js

```diff
@@ -54,6 +57,181 @@ const App = () => {
+              </p>
+            </td>
+            <td>
+              <p className="kaufman__table__title">2</p>
+              <p className="kaufman__table__text">
+                Сосредоточьтесь на каком-то одном навыке.
+              </p>
+            </td>
+            <td>
+              <p className="kaufman__table__title">3</p>
+              <p className="kaufman__table__text">
+                Определите целевой уровень мастерства.
+              </p>
+            </td>
+            <td>
+              <p className="kaufman__table__title">4</p>
+              <p className="kaufman__table__text">
+                Разбейте навык на элементы.
+              </p>
+            </td>
+            <td>
+              <p className="kaufman__table__title">5</p>
+              <p className="kaufman__table__text">
+                Приготовьте всё необходимое для занятий.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <p className="kaufman__table__title">6</p>
+              <p className="kaufman__table__text">
+                Устраните препятствия для занятий.
+              </p>
+            </td>
+            <td>
+              <p className="kaufman__table__title">7</p>
+              <p className="kaufman__table__text">
+                Выделите специальное время для занятий.
+              </p>
+            </td>
+            <td>
+              <p className="kaufman__table__title">8</p>
+              <p className="kaufman__table__text">
+                Создайте быстрые петли обратной связи.
+              </p>
+            </td>
+            <td>
+              <p className="kaufman__table__title">9</p>
+              <p className="kaufman__table__text">
+                Занимайтесь по расписанию, короткими интенсивными интервалами.
+              </p>
+            </td>
+            <td>
+              <p className="kaufman__table__title">10</p>
+              <p className="kaufman__table__text">
+                Уделяйте внимание количеству и скорости.
+              </p>
+            </td>
+          </tr>
+        </table>
+        <img
+          className="kaufman_triangle"
+          src={kaufman_triangle}
+          height="877px"
+          width="877px"
+          alt="Blue Triangle"
+        />
+      </section>
+      <section className="footer">
+        <div className="columns">
+          <div className="column__content">
+            <div className="content__copyright">
+              <img
+                className="footer__logo"
+                src={footerLogo}
+                width="228px"
+                height="32px"
+                alt="logo"
+              />
+              <p className="copyright__author">&copy; Виктор Абанькин</p>
+            </div>
+            <div className="column__content__info">
+              <p className="content__column-heading">О практикуме</p>
+              <div className="column-links">
```

Comment:
Check the [nav tag](https://www.w3schools.com/tags/att_a_target.asp) and use it

Reply:
i think it's fixed? You link goes to  < a > tag

Reply:
Sorry for misleading, here is a [correct one](https://www.w3schools.com/tags/tag_nav.asp)